.gitattributes

```diff
@@ -0,0 +1,7 @@
+/.gitattributes export-ignore
```

Comment:
You can drop the `/` when specifying the files to ignore.

Reply:
@hashar Want to fix this up? :P